ui/vic-ui-h5c/vic/src/vic-webapp/src/app/create-vch-wizard/summary/summary.component.spec.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ Copyright 2017 VMware, Inc. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {ReactiveFormsModule} from '@angular/forms';
+import {ClarityModule} from 'clarity-angular';
+import {HttpModule} from '@angular/http';
+import {SummaryComponent} from './summary.component';
+import {TestScheduler} from 'rxjs/Rx';
+import {Observable} from 'rxjs/Observable';
+
+describe('SummaryComponent', () => {
+
+  let scheduler: TestScheduler;
+  let component: SummaryComponent;
+  let fixture: ComponentFixture<SummaryComponent>;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        ReactiveFormsModule,
+        HttpModule,
+        ClarityModule
+      ],
+      declarations: [
+        SummaryComponent
+      ]
+    });
+  });
+
+  beforeEach(() => {
```

Comment:
Add some comments in this beforeEach to explain what is happening here during each step.